### PR TITLE
Authentication and QueryConfig Controllers

### PR DESCRIPTION
**Pull request by nvmLantana**

### CODE REVIEW

**Review by edward-miller-lcg — CHANGES_REQUESTED** [review]
Can you merge latest from dev? I see some old code in here and I believe there are merge conflicts. Once merged, I can review.

**Comment by nvmLantana** [comment]
> Can you merge latest from dev? I see some old code in here and I believe there are merge conflicts. Once merged, I can review.

Done.

File: DotNet/Shared/Application/Repositories/Interfaces/IPersistenceRepository.cs [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 {
-    void Add(T entity);
-    Task AddAsync(T entity, CancellationToken cancellationToken = default);
-    T Get(string id);
-    Task<T> GetAsync(string id, CancellationToken cancellationToken = default);
-    T Update(T entity);
-    Task<T> UpdateAsync(T entity, CancellationToken cancellationToken = default);
-    void Delete(string id);
-    Task DeleteAsync(string id, CancellationToken cancellationToken = default);
+    public T Add(T entity);
```
Comment (edward-miller-lcg) [inline DotNet/Shared/Application/Repositories/Interfaces/IPersistenceRepository.cs]: I don't think access modifiers are necessary for a interface. Just by being an interface, these methods are public by default.
https://stackoverflow.com/a/6040815

File: DotNet/DataAcquisition/Controllers/AuthenticationConfigController.cs [hunk 2]
```diff
@@ -98,20 +100,25 @@ private async Task SendAudit(string message, string correlationId, string facili
...
-            return BadRequest(bex.Message);
+            await SendAudit(ex.Message, null, facilityId, AuditEventType.Create, null); _logger.LogWarning(ex.Message);
+            return Problem(title: "Bad Request", detail: ex.Message, statusCode: (int)HttpStatusCode.BadRequest);
         }
-        catch (NotFoundException nex)
+        catch (NotFoundException ex)
         {
-            await SendAudit(nex.Message, null, facilityId, AuditEventType.Query, null);
-            return NotFound(nex.Message);
+            await SendAudit(ex.Message, null, facilityId, AuditEventType.Create, null); _logger.LogWarning(ex.Message);
```
Comment (edward-miller-lcg) [inline DotNet/DataAcquisition/Controllers/AuthenticationConfigController.cs]: Pretty sure we don't need these sendAudit messages anymore as our monitoring instrumentation has been improved. @amphillipsLGC can you confirm?

**Review by edward-miller-lcg — CHANGES_REQUESTED** [review]
See additional comments.

**Review by edward-miller-lcg — APPROVED** [review]

**Pull request merged**